client/.env

```diff
@@ -1,2 +1,13 @@
+# ============================================
+# ============================================
+# PRODUCTION CLIENT-SIDE ENVIRONMENT VARIABLES 
+# ============================================
+# ============================================
+#
+#
+# We commit some environment variables to git here 
+# for our production instance to reference. 
+# Head over to `env.local.sample` for more documentation:
+
```

Comment:
Is this enough of an explanation here? To be honest, I'm still at a bit of a loss as to _why_ it's strategic to commit these vars to git, and didn't find very good explanations online...

Reply:
Oh, I think the one advantage of committing them to git is that it makes it easier for developers, especially new contributors.

With the tenant platform, for instance, which has nothing like this, the disadvantage is that new contributors either need to sign up for a new account on every service they want to integrate with, or they need to get a copy of the `.env` file from another developer--which can be especially frustrating if the contributor is an open-source volunteer and doesn't _know_ any other developers on the project.

On the other hand, if the project can just have some default values for the environment variables without introducing any security holes, this can help new developers hit the ground running much quicker.

I guess another advantage is that, if these values are the same ones used in production, it can lead to better [dev/prod parity](https://12factor.net/dev-prod-parity) too.

Reply:
It might also help to point the reader to CRA's documentation on adding custom environment variables too, in part for their own edification but also to help convince them that we're not just being weird: https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
Err also it might be worth calling this file "DEFAULT CLIENT-SIDE ENVIRONMENT VARIABLES" rather than production?  I'm not sure what terminology CRA uses, actually, but as I mentioned in an earlier comment, I think the biggest benefit comes from these being useful development defaults, rather than necessarily production values (I might be wrong but I think that it might be possible to override these values with actual environment variables we set through e.g. Heroku, and in some cases we might end up doing that).

Reply:
Ok great! This definitely makes more sense now. Also— this helps explain too why the one var we commit to git is precisely the one "required" env var we have on on the client side. All the other vars are optional, which means that a new contributor wouldn't need to do anything with those to get the codebase up and running.